### DISCUSSION TITLE
Default plan disappeared

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#1034] [Frontier TestNet - Bug Reporting]
I appreciate this is being reported after the fact, but my default plan disappeared at some point over the last few days. Is there a scenario where this could happen? If not, is there a log to know exactly when it was removed? I did upgrade project indexer and query versions to 1.0 but other than that my indexer has been stable.

My deployment specific plans are intact and I've been able to create new default plans.

**Comment by mzxyz** [comment]
@lilian79511
